src/components/datetime/DateTimeController.cpp

```diff
@@ -117,4 +117,30 @@ char const* DateTime::MonthsString[] = {"--", "JAN", "FEB", "MAR", "APR", "MAY",
 char const* DateTime::MonthsStringLow[] = {"--", "Jan", "Feb", "Mar", "Apr", "May", "Jun", "Jul", "Aug", "Sep", "Oct", "Nov", "Dec"};
 
 char const* DateTime::MonthsLow[] = {
-  "--", "January", "February", "March", "April", "May", "June", "July", "August", "September", "October", "November", "December"};
\ No newline at end of file
+  "--", "January", "February", "March", "April", "May", "June", "July", "August", "September", "October", "November", "December"};
+
+void DateTime::GetTimeStr(char* string, bool is24h) {
```

Comment:
It's a personal preference but I dislike using a `bool` as a stand-in for a two state enum. Making the second argument a `ClockType` instead would be better IMO. This would actually make the calling code simpler too since they all end up doing a comparison like `someClockTypeVariable == ClockType::H24` now anyways.